### PR TITLE
codegen: enable cross-directory actor inherits

### PR DESCRIPTION
**Pull request by goldenleaves**

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -116,12 +116,10 @@ seastar::future<hiactor::Integer> bank_account::check() {
 Note that there are some rules which should be followed when defining an actor class (in `actor_name.act.h`):
 - A customized actor class must be defined in a c++ header file with the suffix `.act.h`.
 - The annotation `ANNOTATION(actor:impl)` must be specified in actor class definition.
-- A customized actor class must be derived from the template `actor` class, the `set_max_concurrency` method
+- A customized actor class must be derived from the template `hiactor::actor` class, the `set_max_concurrency` method
```
Comment (LiSu) [inline docs/tutorial.md]: template --> base

File: docs/tutorial.md [hunk 2]
```diff
@@ -179,15 +177,11 @@ include ($install_prefix/hiactor_codegen/ActorAutoGen.cmake)
 # actor codegen cmake func
 # @param "actor-autogen": specify the cmake target for actor codegen.
 # @param "actor_gen_files": specify the generated actor definition files.
-# @param "ACTOR_SOURCE_DIR": set the actor source dir that contains customized actor files.
-# @param "HIACTOR_INCLUDE": set the hiactor include dir.
-# @param "SYSTEM_INCLUDE": set other system include dirs with a semicolon-separated list if used by your project.
-# @param "USER_INCLUDE": set the current project include dirs with a semicolon-separated list.
+# @param "SOURCE_DIR": set the source dir that contains customized actor files.
```
Comment (LiSu) [inline docs/tutorial.md]: customized --> user-defined

**Review by LiSu — APPROVED** [review]

File: docs/tutorial.md (hunk 2 above)
Comment (goldenleaves) [reply docs/tutorial.md]: ok

File: docs/tutorial.md (hunk 1 above)
Comment (goldenleaves) [reply docs/tutorial.md]: ok

**Pull request merged**